VmAgent.Core/MultiLogger.cs

```diff
@@ -13,62 +13,74 @@ namespace Microsoft.Azure.Gaming.VmAgent.Core
     public class MultiLogger : ILogger
     {
         private readonly ILogger _logger;
-        private readonly TelemetryClient _genevaTelemetryClient;
+		private readonly TelemetryClient _genevaTelemetryClient;
+		private ILogger _genevaOTelLogger;
+        private const string VmAgentLoggerName = "VmAgentLogger";
 
-        public MultiLogger(ILogger logger, TelemetryClient genevaTelemetryClient)
+		public MultiLogger(ILogger logger, TelemetryClient genevaTelemetryClient, ILogger genevaLogger = null)
```

Comment:
>MultiLogger [](http://example.com/codeflow?start=9&length=11)

We should add comments on why we have 2 ILogger parameters here.

Are we going to remove/replace the existing ILogger in the future?